opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -432,16 +433,20 @@ namespace Opm {
                 if (FluidSystem::phaseIsActive(FluidSystem::oilPhaseIdx)) {
                     saturationsOld[FluidSystem::oilPhaseIdx] = oilSaturationOld;
                 }
-
-                Scalar tmp = pressureNew - pressureOld;
-                resultDelta += tmp*tmp;
-                resultDenom += pressureNew*pressureNew;
```

Comment:
is this unused? if yes, good catch!

Reply:
This code need to be changed, but will change results. I will like that the pull request for facilitating onephase will be merged first. Else a lot of testing has to be done for all relevant cases...